### DISCUSSION TITLE
unable to build kubernetes objects from release manifests

### DISCUSSION THREAD

**Discussion by saleem-unifycare** [hystax/optscale#78] [Q&A] [answered]
HI Team,
we are getting below error when running  "./runkube.py --use-socket --with-elk  -o overlay/user_template.yml -- <deployment name> component_versions.yaml"

`Release "optscale" does not exist. Installing it now.
Error: unable to build kubernetes objects from release manifest: [resource mapping not found for name: "mongo" namespace: "" from "": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "rabbitmq" namespace: "" from "": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "bi-scheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "booking-observer-scheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "calendar-observer-scheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "cleaninfluxdb" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "demoorgcleanup" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "failed-imports-dataset-generator" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "insider-scheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "metroculusscheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "organization-violations-scheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "report-import-scheduler-0" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "report-import-scheduler-1" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "report-import-scheduler-24" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "report-import-scheduler-6" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "resource-discovery-scheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "resource-observer-scheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "resource-violations-scheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "thanos-compactor" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "trapper-scheduler" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "users-dataset-generator" namespace: "" from "": no matches for kind "CronJob" in version "batch/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "acura" namespace: "default" from "": no matches for kind "Ingress" in version "extensions/v1beta1"
ensure CRDs are installed first]
Traceback (most recent call last):
  File "/home/ubuntu/optscale/optscale-deploy/./runkube.py", line 388, in <module>
    acr.start(args.check, args.update_only)
  File "/home/ubuntu/optscale/optscale-deploy/./runkube.py", line 316, in start
    subprocess.run(update_cmd.split(), check=True)
  File "/usr/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['helm', 'upgrade', '--install', '-f', 'tmp/base_overlay', '-f', 'overlay/user_template.yml', 'optscale', 'optscale']' returned non-zero exit status 1.`

please guide us.

**Comment by stanfra** [comment, marked as answer]
Hi @saleem-unifycare, now we use differend deployment scheme as earlier. Pleae try to deploy cluster using README.md from optscale repo.
